modules/alloy/src/main/ts/ephox/alloy/behaviour/disabling/DisableState.ts

```diff
@@ -1,15 +1,15 @@
-import { Cell } from '@ephox/katamari';
+import { Singleton } from '@ephox/katamari';
 
 import { nuState } from '../common/BehaviourState';
 import { DisableState } from './DisableTypes';
 
 const init = (): DisableState => {
-  const lastDisabledState = Cell(false);
+  const lastDisabledState = Singleton.value<boolean>();
```

Comment:
Why is this now a singleton can this be `disabled/enabled/<unknown>` ? It's either disabled or enabled it can't really be in a third state?